UI/window-basic-main-outputs.cpp

```diff
@@ -1786,6 +1786,10 @@ inline void AdvancedOutput::SetupRecording()
 	unsigned int cy = 0;
 	int idx = 0;
 
+	/* Hack to allow recordings without any audio tracks selected. It is no
+	 * longer possible to select such a configuration in settings, but legacy
+	 * configurations might still have this configured and we don't want to
+	 * just break them. */
```

Comment:
How long has this been possible? Wondering if there's a way for us to force the user to select a track by preventing an output from starting in this configuration, as a future approach.